packages/cicada/build.sh

```diff
@@ -0,0 +1,24 @@
+TERMUX_PKG_HOMEPAGE=https://github.com/mitnk/cicada
+TERMUX_PKG_DESCRIPTION="A bash like Unix shell"
+TERMUX_PKG_MAINTAINER="Hugo Wang <w@mitnk.com>"
+TERMUX_PKG_LICENSE="MIT"
+TERMUX_PKG_VERSION=0.9.11
+TERMUX_PKG_SRCURL=https://github.com/mitnk/cicada/archive/v$TERMUX_PKG_VERSION.tar.gz
+TERMUX_PKG_SHA256=a3755aab5c60887587e2e98d0786e7cbf03b9871e931d2a6a177aa093427253d
+TERMUX_PKG_BUILD_IN_SRC=true
+
+termux_step_pre_configure() {
+    rm -f Makefile
+}
+
+termux_step_post_make_install() {
+    if [ "$TERMUX_ON_DEVICE_BUILD" = "true" ]; then
+        if [ ! -f "$HOME/.inputrc" ]; then
+            cp -v misc/inputrc $HOME/.inputrc
```

Comment:
Package shouldn't preinstall or overwrite anything in `$HOME`, especially if configuration file belongs to different package (in this case `readline`).

Reply:
Ah, ok. Make sense to me. Will update it.

Reply:
updated.